src/Hystrix.Dotnet/DateTimeProvider.cs

```diff
@@ -2,11 +2,9 @@
 
 namespace Hystrix.Dotnet
 {
-    public class DateTimeProvider
+    public class DateTimeProvider : IDateTimeProvider
     {
-        public virtual long GetCurrentTimeInMilliseconds()
-        {
-            return DateTime.UtcNow.Ticks / TimeSpan.TicksPerMillisecond;
-        }
+        public long CurrentTimeInMilliseconds =>
+            DateTime.UtcNow.Ticks / TimeSpan.TicksPerMillisecond;
```

Comment:
Changed this from a method to a property, just to be consistent with .NET and other date time libraries.